ShopsTable.API/Controllers/ShopController.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using ShopsTable.API.Models;
+using ShopsTable.API.Services.Contracts;
+
+namespace ShopsTable.API.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class ShopController : ControllerBase
+    {
+
+        private readonly ILogger<ShopController> _logger;
+        private readonly IShopService _shopService;
+
+        public ShopController(ILogger<ShopController> logger,
+            IShopService shopService)
+        {
+            _logger = logger;
+            _shopService = shopService;
+        }
+
+        [HttpPost]
+        public async Task<Shop> CreateAsync([FromBody] Shop shop)
+        {
+            await _shopService.CreateAsync(shop);
+            return shop;
+        }
+
+        [HttpGet("{id}")]
+        public Shop Get(Guid id)
```

Comment:
async Task<IActionResult> - all endpoints shous return it

Reply:
[ProducesResponseType(StatusCodes.Status200OK, Type = typeof(Shop))]
[ProducesResponseType(StatusCodes.Status404NotFound)]

Add such attributes for all endpoints for all controllers